pipeline/load.py

```diff
@@ -272,7 +289,7 @@ def insert_single_sale(cursor: DBCursor, single_sale: Dict[str, Any]) -> None:
 
     for tag in single_sale["track_tags"]:
         if tag:
-            get_or_insert_tags(cursor, tag, track_id=track_id)
+            get_or_insert_tags_and_assignments(cursor, tag, track_id=track_id)
```

Comment:
could this just be `track_id`

Reply:
This function is used for both albums and tracks, and so can take in both album_id and track_id as arguments. Both these arguments are `None` by default, and the specific target of the function is specified by mentioning the specific argument, such as `track_id=track_id`. This method of specifying the argument is used when there are more than one arguments with default values. 
You can read more about this [here](https://stackoverflow.com/questions/25374814/multiple-default-values-for-function-arguments-python)

Reply:
That makes sense! Good explanation